docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -99,40 +96,1064 @@ Dropped columns can be provided to the tool in the form of insert statements int
+
+consume_partition_start(ps)
+"""""""""""""""""""""""""""
+
+Part of the Consume API. Called on the start of each partition. Parameter is of type `Scylla.partition_start <scylla-partition-start-type_>`_.
+Returns whether to stop. If ``true``, `consume_partition_end() <scylla-consume-partition-end-method_>`_ is called, skipping the content of the partition. If ``false``, consumption follows with the content of the partition.
+
+consume_static_row(sr)
+""""""""""""""""""""""
+
+Part of the Consume API. Called if the partition has a static row. Parameter is of type `Scylla.static_row <scylla-static-row-type_>`_.
+Returns whether to stop. If ``true``, `consume_partition_end() <scylla-consume-partition-end-method_>`_ is called, skipping the remaining content of the partition. If ``false``, consumption follows with the remaining content of the partition.
+
+consume_clustering_row(cr)
+""""""""""""""""""""""""""
+
+Part of the Consume API. Called for each clustering row. Parameter is of type `Scylla.clustering_row <scylla-clustering-row-type_>`_.
+Returns whether to stop. If ``true``, `consume_partition_end() <scylla-consume-partition-end-method_>`_ is called, skipping the remaining content of the partition. If ``false``, consumption follows with the remaining content of the partition.
+
+consume_range_tombstone_change(crt)
+"""""""""""""""""""""""""""""""""""
+
+Part of the Consume API. Called for each range tombstone change. Parameter is of type `Scylla.range_tombstone_change <scylla-range-tombstone-change-type_>`_.
+Returns whether to stop. If ``true``, `consume_partition_end() <scylla-consume-partition-end-method_>`_ is called, skipping the remaining content of the partition. If ``false``, consumption follows with the remaining content of the partition.
+
+.. _scylla-consume-partition-end-method:
+
+consume_partition_end()
+"""""""""""""""""""""""
+
+Part of the Consume API. Called at the end of the partition.
+Returns whether to stop. If ``true``, `consume_sstable_end() <scylla-consume-sstable-end-method_>`_ is called, skipping the remaining content of the sstable. If ``false``, consumption follows with the remaining content of the sstable.
+
+.. _scylla-consume-sstable-end-method:
+
+consume_sstable_end()
+"""""""""""""""""""""
+
+Part of the Consume API. Called at the end of the sstable.
+Returns whether to stop. If true, `consume_stream_end() <scylla-consume-stream-end-method_>`_ is called, skipping the remaining content of the stream. If false, consumption follows with the remaining content of the stream.
+
+.. _scylla-consume-stream-end-method:
+
+consume_stream_end()
+""""""""""""""""""""
+
+Part of the Consume API. Called at the very end of the stream.
+
+Scylla LUA API
+~~~~~~~~~~~~~~
+
+In addition to the `Scylla Consume API <scylla-consume-api_>`_, the Lua bindings expose various types and methods to allow for working with Scylla types and values.
+These are above those that the base, string and table Lua standard libraries expose.
```

Comment:
> These are above those

@denesb 
I'm not sure what that means. This sentence needs some rewording, but I'm not sure about the meaning. I think it could be:
"These types and methods have priority over the types and methods exposed by the base, string, and table Lua standard libraries."
OR
"These types and methods should be preferred over the types and methods exposed by the base, string, and table Lua standard libraries."